dts/common/nordic/nrf54l15.dtsi

```diff
@@ -620,6 +620,31 @@
 				status = "disabled";
 			};
 
+			power: power@10e000 {
+				compatible = "nordic,nrf-power";
+				reg = <0x10e000 0x1000>;
+				interrupts = <270 NRF_DEFAULT_IRQ_PRIORITY>;
+				status = "disabled";
+				#address-cells = <1>;
+				#size-cells = <1>;
+
+				gpregret1: gpregret1@51c {
+					#address-cells = <1>;
+					#size-cells = <1>;
+					compatible = "nordic,nrf-gpregret";
+					reg = <0x51c 0x1>;
```

Comment:
`ranges = <0x0 0x10e000 0x1000>;` needs to be added in the parent node for this offset to be handled properly.

Reply:
added